docs/public/SUMMARY.md

```diff
@@ -0,0 +1,110 @@
+- [Installation]()
+    - [Prerequisites](installation/prerequisites.md)
+    - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
+    - [Zonemaster-Engine](installation/zonemaster-engine.md)
+    - [Zonemaster-CLI](installation/zonemaster-cli.md)
+    - [Zonemaster-Backend](installation/zonemaster-backend.md)
+    - [Zonemaster-GUI](installation/zonemaster-gui.md)
+    - [Docker](installation/docker.md)
+- [Upgrade]()
+    - [Backend](upgrade/backend.md)
+- [Configuration]()
+    - [Profiles](configuration/profiles.md)
+    - [Backend](configuration/backend.md)
+        - [Environment Variables](configuration/backend-environment-variables.md)
+    - [GUI](configuration/gui.md)
+- [Using]()
+    - [CLI](using/cli.md)
+    - [RPCAPI](using/getting-started.md)
+        - [RPCAPI Reference](using/backend-api.md)
+    - [Telemetry](using/telemetry.md)
+    - [GUI](using/gui-api.md)
+- [Specifications](specifications/tests/README.md)
```

Comment:
I see no reference to `specifications/test-types/*` in the ToC. Has it been omitted on purpose? If yes, why is it kept in the public documentation?

Reply:
What you find here under "Specification" is different from "docs/public)/specifications/". I find that confusing.

And also, is the "specification/tests/README.md" introduction to the other documents? I find it hard to navigate.

Reply:
> I find that confusing.
> And also, is the "specification/tests/README.md" introduction to the other documents? I find it hard to navigate.

What would you be expecting so that it would be less confusing?